src/controllers/sign.out.controller.ts

```diff
@@ -25,7 +27,7 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     }
 
     if (req.body["sign_out"] === 'yes') {
-      return res.redirect(config.ACCOUNTS_SIGN_OUT_URL);
+      return res.redirect(config.UPDATE_ACCOUNTS_SIGN_OUT_URL);
```

Comment:
Should this be changed if its used in the register journey?

Reply:
The variable `UPDATE_ACCOUNTS_SIGN_OUT_URL` doesn't seem to exist?

Reply:
This should have been, was missing from the commit. Its there now.